### PR TITLE
Fix instructions for enabling JSON-P for Health-Checks.

### PR DESCRIPTION
**Pull request by michael-simons**

### CODE REVIEW

**Comment by michael-simons** [comment]
Just sent out the OCA.

File: docs/se/health/01_health.adoc [hunk 1]
```diff
@@ -43,10 +43,15 @@ A typical health check combines the statuses of all the dependencies that
 
 === Prerequisites
 
-Declare the following dependency in your project:
+Declare the following dependencies in your project:
```
Comment (tomas-langer) [inline docs/se/health/01_health.adoc:46]: This is not needed - `helidon-health` transitively depends on `helidon-media-jsonp`

File: docs/se/health/01_health.adoc [hunk 2]
```diff
@@ -139,24 +144,33 @@ NOTE: Health check requires the `JSON-P` support to be enabled. See the example
 
 === Example
 
+[source,java]
+.Enable JSON-P support on the server
```
Comment (tomas-langer) [inline docs/se/health/01_health.adoc:148]: This is also not needed - `HealthSupport` uses the JSON-P writer directly not to interfere with user's application choice of media types.

**Review by tomas-langer — CHANGES_REQUESTED** [review]

**Comment by tomas-langer** [comment]
Hi @michael-simons, thanks for your contribution. The `HealthSupport` in current version does not require explicit registration of `JsonpSupport` with `WebServer`. You have identified an error in our documentation, the correct (new) approach is actually simpler than both the original and your updated docs. Detail in review comments

**Comment by michael-simons** [comment]
A nice. I was wondering, especially about enabling the support. I didn't have this in my examples and things have been working nevertheless. I fixed my PR.

**Review by tomas-langer — APPROVED** [review]

**Pull request merged**